### PR TITLE
Add license to each file

### PR DESCRIPTION
**Pull request by kemingy**
tool: https://github.com/google/addlicense

### CODE REVIEW

**Comment by kemingy** [comment]
This should be merged before #134. Otherwise, there will be some conflicts.

File: examples/distil_bert_server_pytorch.py [hunk 1]
```diff
@@ -1,3 +1,17 @@
+# Copyright 2022 MOSEC
```
Comment (lkevinzc) [inline examples/distil_bert_server_pytorch.py:1]: Usually I saw the format be like `Copyright 2022 MOSEC Authors. All Rights Reserved.` for the first line?

Comment (kemingy) [reply examples/distil_bert_server_pytorch.py:1]: This is auto-generated by the tool `addlicense`.

Check other projects:
* https://github.com/kubernetes/kubernetes/blob/master/cmd/kubectl/kubectl.go
* https://github.com/falconry/falcon/blob/master/falcon/routing/compiled.py

It seems they don't have `All rights reserved. BTW, I don't really know the legal difference.

Comment (lkevinzc) [reply examples/distil_bert_server_pytorch.py:1]: An example: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/framework/allocator.h#L1

Anyway, I think the exact format is not particularly important. However, `MOSEC` is the project name, at least we need to change it to `MOSEC Authors`.

Comment (kemingy) [reply examples/distil_bert_server_pytorch.py]: Fixed

**Review by lkevinzc — APPROVED** [review]

**Pull request merged**